### DISCUSSION TITLE
乱码？

### DISCUSSION THREAD

**Discussion by yuyi2439** [howmanybots/onebot#103] [问答] [answered]
![image](https://user-images.githubusercontent.com/68320855/116814103-b7f3a780-ab89-11eb-898b-2b60ad7fa34f.png)
我用易语言做的，易语言和onebot里显示不一样，是不是两个软件（就这样叫吧）编码不同？

**Comment by yuyi2439** [comment]
编辑框1的内容是“有人吗”

**Comment by ExerciseBook** [comment, marked as answer]
UTF-8 请。

**Reply by yuyi2439** [reply]
我在网上找了很多教程都没有用
关于易语言编码，实在是找不清

**Reply by ExerciseBook** [reply]
精易有 编码_ansi到UTF8 和 编码_UTF8到Ansi。
你发送的时候用 ansi到UTF8，你接收的时候用 UTF8到Ansi。

虽然精易这个很垃圾，但是至少能用。。。。

**Reply by yuyi2439** [reply]
但是还要转成文本型再发送，但是转过之后又变成乱码了

**Reply by ExerciseBook** [reply]
那你为什么不考虑一下 `编码_URL编码(编码_ansi到UTF8("你好"))` 呢？

**Reply by yuyi2439** [reply]
成功了！感谢！
不过下面这种解决方案可能会更好
编码_URL编码 (编辑框1.内容, , 真)

**Reply by ExerciseBook** [reply]
搞定了就好。

**Reply by anye-star** [reply]
请求参数需要urlencode，或者整个请求体转utf8编码后字节集提交
